src/database/migrations/versions/41cdafa531cf_temp_migration.py

```
@@ -0,0 +1,36 @@
+"""temp_migration
```

Pylint (3.3.7)

[convention] Module name "41cdafa531cf_temp_migration" doesn't conform to snake_case naming style

(C0103)